### PR TITLE
Setup protected routes

### PR DESCRIPTION
**Pull request by mattsre**
Users now have to login to proceed to any pages other than those listed by the "openRoutes" variable.

### CODE REVIEW

**Comment by mattsre** [comment]
Closed pull request after noticing better way of doing something. Will reopen shortly.

**Review by ArowShot — APPROVED** [review]

File: kite-api/src/resolvers/user.resolver.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ const UserResolvers = {
       return `${err}`;
     }
   },
+  getUserScope: async (_, req) => {
```
Comment (ianfabs) [inline kite-api/src/resolvers/user.resolver.js:26]: What is that underscore?

**Review by ianfabs — APPROVED** [review]
Wheel, Snipe, Celly bud!

File: kite-api/src/resolvers/user.resolver.js (hunk 1 above)
Comment (ianfabs) [reply kite-api/src/resolvers/user.resolver.js:26]: 🤢

Comment (ianfabs) [reply kite-api/src/resolvers/user.resolver.js:26]: Nevermind, you just explained it to me

**Pull request merged**